Add "run_time" and "simulation_time" to automatic timestamp conversion fields (#2226)
cognitedata/cognite-sdk-python: CHANGELOG.md

- `Fixed` for any bug fixes.
- `Security` in case of vulnerabilities.

> ## Unreleased

Reviewer: Good QOL addition! Let's bump patch version for this 👍

Author: I updated the CHANGELOG and bumped the version in the `pyproject.toml` and `_version.py`

> ### Added
> - New time attributes `run_time` and `simulation_time` to the list of fields that get automatically converted to timestamp format when converted to DataFrames

Reviewer: ```suggestion
### Changed
- Attributes `run_time` and `simulation_time` are now automatically converted to timestamp format (when calling `to_pandas(...)`)
```
